bin/authorize-cicd

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [[ -z $1 || -z $2 ]]
+then
+    echo "USAGE: $0 [user] [authlist]"
+    exit 1
+fi
+
+authlist=$(cat $2 | grep user | sed 's#.*: \(.*$\)#\1#')
+for user in $authlist
+do
+    if [[ $user == $1 ]]
+    then
+        echo "SUCCESS: User '$1' authorized"
+        exit 0
+    fi
+done
+
+echo "FAILED: User '$1' not authorized."
```

Comment:
Can we say, user account is not enabled for code merges. Please contact maintainers?
The word `authorize` can make first time contributors worry.

Reply:
This wording was reviewed on core and then replicated out to the other repos. I'd rather keep it standard. Most users are just going to see that they didn't pass something from the github interface and probably not know to dig into jenkins logs.